code/modules/paperwork/records/info/waw.dm

```diff
@@ -501,8 +501,9 @@
 	Qliphoth Counter : 1	<br>
 	Work Damage Type : White	<br>
 	Work Damage : High	<br>
-	- When the work result was Normal, the Qliphoth Counter lowered with a normal probability.	<br>
+	- When the work result was Bad, the Qliphoth Counter lowered with a high probability.	<br>
 	- When the work result was Good, the Qliphoth Counter lowered with a high probability.	<br>
+	- WARNING: Upon O-01-17's death, immediately evacuate the area surrounding O-01-17's corpse.	<br>
```

Comment:
This sounds like it sets off a nuke. Just say that it deals red damage around it on death.